### PR TITLE
Fix repmgrd service on Ubuntu

### PR DESCRIPTION
**Pull request by to-bar**
- Set 'daemonize=false' option in /etc/default/repmgrd
- Add test for 'repmgr service status'
- Update postgres user in sudoers file

### CODE REVIEW

**Comment by przemyslavic** [comment]
Tested on AWS Ubuntu:
```
   ID | Name                                              | Role    | Status    | Upstream                                          | repmgrd     | PID | Paused? | Upstream last seen
----+---------------------------------------------------+---------+-----------+---------------------------------------------------+-------------+-----+---------+--------------------
 1  | ec2-1-1-1-1.eu-west-1.compute.amazonaws.com | primary | * running |                                                   | not running | n/a | n/a     | n/a                
 2  | ec2-2-2-2-2.eu-west-1.compute.amazonaws.com  | standby |   running | ec2-1-1-1-1.eu-west-1.compute.amazonaws.com | not running | n/a | n/a     | n/a                
```

repmgr is not running :( (tests failed)

File: core/src/epicli/data/common/ansible/playbooks/roles/postgresql/tasks/extensions/replication/extension.yml [hunk 1]
```diff
@@ -18,8 +18,7 @@
       /bin/systemctl restart {{ pg.service_name[ansible_os_family] }},
       /bin/systemctl reload {{ pg.service_name[ansible_os_family] }},
       /bin/systemctl start {{ repmgr.service_name[ansible_os_family] }},
-      /bin/systemctl stop {{ repmgr.service_name[ansible_os_family] }},
-      {{ repmgr.bin_dir[ansible_os_family] }}/repmgr standby follow
```
Comment (atsikham) [inline core/src/epicli/data/common/ansible/playbooks/roles/postgresql/tasks/extensions/replication/extension.yml:22]: ❤️

File: core/src/epicli/data/common/ansible/playbooks/roles/postgresql/tasks/upgrade/extensions/replication/repmgr-upgrade.yml [hunk 2]
```diff
@@ -68,6 +68,28 @@
   module_defaults:
     yum: { lock_timeout: "{{ yum_lock_timeout }}" }
 
+- name: repmgr for PG {{ pg_version }} | Update postgres user in sudoers file
+  lineinfile:
+    path: /etc/sudoers
+    regexp: "^postgres ALL=\\(ALL:ALL\\) NOPASSWD:"
```
Comment (atsikham) [inline core/src/epicli/data/common/ansible/playbooks/roles/postgresql/tasks/upgrade/extensions/replication/repmgr-upgrade.yml:74]: ❤️

**Review by atsikham — APPROVED** [review]
LGTM

**Review by przemyslavic — APPROVED** [review]

**Review by erzetpe — APPROVED** [review]

**Pull request merged**